### ISSUE TITLE
Expose way to do key combinations

### ISSUE THREAD

**Issue by EgosOwn** [yunginnanet/sendkeys#1]
I'm using this in a project where i need to be able to do key combinations and i don't think this lib supports that.

I may contribute a feature for this

**Comment by yunginnanet** [comment]
Good idea! If I have time, I'll take a look. If not, pull requests are always welcome :)
